### DISCUSSION TITLE
adding .jpg graphics to tailwind

### DISCUSSION THREAD

**Discussion by DaveGrace** [tailwindlabs/tailwindcss#2487] [Help] [answered]
Hi, 

I want to add some .jpg files to my tailwind.  I have them stored in a graphics folder in my redwood src folder.

This is the code for the graphic:

      `import { Link, routes } from '@redwoodjs/router'
const AboutPage = () =>{
  return(

<>
<nav className="bg-red-500 shadow">
 <img className="h-10 w-full" src="D:\redwoodPlay\redwoodtailwind\web\src\graphics\Broulee island.jpg"></img>

  <div className="max-w-7xl mx-auto px-5 sm:px-6 lg:px-8">
    <div className="relative flex justify-between h-15">
      <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">`

It does not load (I get a broken file icon), and the chrome devtools says that I am not allowed to load a local resource.

Can you tell me what I need to do to fix this?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Are you able to share a code reproduction of your setup in a GitHub repo?

Looks like your code snippet wasn't rendered properly either, so it's a bit hard to help troubleshoot based on this.

If you can share your setup, we'll be able to have a look for you 👍

**Reply by DaveGrace** [reply]
thanks I worked it out.  A beginners mistake - I was referring to my computers hard disk, which is not accessible on the server, rather than putting the graphic in the folders that have been uploaded to it.  Now that the file is in the right place, there is no problem!!

Thanks for offering to help me.

**Reply by simonswiss** [reply]
Haha I think we’ll all been there once!

No problem - glad you got it sorted 🎉
